coalib/parsing/DefaultArgParser.py

```diff
@@ -231,6 +231,10 @@ def default_arg_parser(formatter_class=None):
         help="don't create .orig backup files before patching")
 
     misc_group.add_argument(
+        '-res', '--apply-patch', const=True, action='store_const',
```

Comment:
Euhm, doesn't this conflict with `-a`, `--apply-patches`? I think it's confusing, I suggest `--single-action`, `-A`